bindings/go-apiv1/.gitignore

```diff
@@ -0,0 +1,3 @@
+#SPDX-License-Identifier: Apache-2.0
```

Comment:
Are IDE gitignores required here?
The go-apiv1 and goapiv2 directories are templates for the generated modules published to https://github.com/hyperledger/fabric-protos-go and https://github.com/hyperledger/fabric-protos-go-apiv2

Reply:
done in the same way as all fabric repositories....

Reply:
Sometimes I open these repositories through ide (for debugging) and ide adds its service files and shows that there were changes in the repository. And they really haven't. Having a .gitignore file will help to see if something major has changed.

Reply:
I guess James' point is that these directories are pushed directly to the published repositories, which currently do not have (or need) gitignore files, because nobody should be editing them directly.  All changes should happen in this repository. The gitignore at the root of this repository already has the entries you are adding here so this repository is already covered.

Reply:
When I go to debug some functionality, I connect fabric-protos-go-apiv2 or fabric-protos-go repositories directly (via replace) and make changes or not on the fly. The lack of a .gitignore file is very annoying in this case.

Reply:
In general I don't insist, maybe in debugging I'm not working with these repositories correctly. 

Actually, I need to add the .gitignore file to the fabric-protos-go-apiv2 and fabric-protos-go, but this is possible only through the current repository.
So far I am not comfortable without these files.

Reply:
Since this repository is where changes need to be delivered, a better approach might be to generate and modify the bindings here, then replace from this location instead.

Having said that, if it's solving a real problem for you and not breaking anything, fair enough I guess.

Reply:
There's no harm in adding them if it's useful, which it sounds like it could be for debugging. My only slight concern is that they might lead people to believe that they should be making changes in the published repos, which they definitely shouldn't.

Reply:
> My only slight concern is that they might lead people to believe that they should be making changes in the published repos, which they definitely shouldn't.

It seems to me that it should be in the documentation. 
Or in the description of the repositories themselves fabric-protos-go-apiv2 and fabric-protos-go.

I can in README.md, add the appropriate entry

Reply:
I think the only place it's mentioned at the moment is the [README.md file above the individual bindings](https://github.com/hyperledger/fabric-protos/blob/main/bindings/README.md). Adding something to the individual readmes sounds helpful, although maybe that would be better as a separate PR to avoid derailing this one too much? I don't mind either way though.

Reply:
I agree, I'll split it into different pr